CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ ___Note:__ Yet to be released changes appear here._
 
 ### General
 
+* `FEAT`: automatically download Camunda connectors ([#4302](https://github.com/camunda/camunda-modeler/issues/4302))
```

Comment:
Let's move this to "unreleased", as we did not communicate it in any way or form.

Reply:
Done

Reply:
:rocket: